src/models/apifilter.ts

```diff
@@ -11,8 +11,18 @@ export class ApiFilter {
       return '';
     }
 
-    public addFilter(name: string, value: string): void {
+    public addFilter(name: string, value: string, unique: boolean = false): void {
```

Comment:
Is it better to have:
 - unique as an optional parameter _addFilter(name, value, unique)_
or 
- a new method _addUniqueFilter(name, value)_

Reply:
Both approaches are backward compatible